neurobooth_os/config.py

```diff
@@ -33,7 +35,38 @@ def validate_folder(value: str) -> None:
         raise IOError(f"The path '{value}' is not a folder.")
 
 
+class DatabaseSpec(BaseModel):
+    dbname: str
+    user: str
+    password: Annotated[str, Field(alias='pass')]
```

Comment:
What is the purpose the alias annotation?

Reply:
All the aliases are to avoid needing to change the existing config files. They fall into two categories:
1. "pass" is a keyword, so I needed a valid variable name
2. The other aliases are for naming consistency in the model